src/javascripts/ng-admin/Crud/misc/EntryFormatter.js

```diff
@@ -75,6 +75,13 @@ export default class EntryFormatter {
                         value: entry.listValues[field.name()].join(', ')
                     };
                 };
+            case 'embedded_list':
+                return function (entry) {
+                    return {
+                        name: label,
+                        value: entry.values[field.name()].map((obj => obj[field.targetFields()[0].name()]))
```

Comment:
Perhaps would it be more coherent to use the same value as for the `reference_many`?

```
value: entry.listValues[field.name()].join(', ')
```